receipt_label/receipt_label/langchain/nodes/phase2.py

```
@@ -0,0 +1,207 @@
+
+    print(f"   Creating {len(line_totals)} parallel Phase 2 tasks")
+
+    words: List[ReceiptWord] = state.words
+    words_by_line: dict[int, List[str]] = {}
+    try:
+        for w in words:
+            words_by_line.setdefault(w.line_id, []).append(
+                getattr(w, "text", "")
+            )
+    except Exception:
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)